pkgs/top-level/all-packages.nix

```diff
@@ -19401,7 +19401,9 @@ with pkgs;
 
   todoist = callPackage ../applications/misc/todoist { };
 
-  todoist-electron = callPackage ../applications/misc/todoist-electron { };
+  todoist-electron = callPackage ../applications/misc/todoist-electron {
+    electron = electron_24;
```

Comment:
I'm wondering, should this be unpinned and let it use whatever electron defaults to?